src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -609,6 +609,7 @@ public static IEnumerable<object> GetAsync_Chunked_VaryingSizeChunks_ReceivedCor
             });
         }
 
+        [ActiveIssue(29802, TargetFrameworkMonikers.Uap)]
```

Comment:
Should we just say that UWP does not have this behavior and will not have it? It is the platform behavior and we didn't change it in long time ...
Disabled test would mean we have intent to fix it in UWP.

@davidsh thoughts?

Reply:
I disabled it against the general "lots of hanging / failing tests on UWP" issue.  If we're keeping the specialized handler, then presumably we'd want to fix this (otherwise why did we fix it in SocketsHttpHandler).  If we're not, then it would automatically be fixed by moving to SocketsHttpHandler.

Reply:
Fair, we can nuke it once we migrate to SocketsHttpHandler in UWP too.

Reply:
I would have preferred we just use SkipOnTargetFramework(Uap) for this.  I plan to close that "" issue once I work thru the rest of the tests to determine what needs to be fixed vs. what can be skipped (because we don't care).  And yes, once SocketsHttpHandler replaces the current UAP HTTP stack, then we can revisit the SkipOnTargetFramework tests.